sandy/sections/mf35.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+"""
+This module contains only one public functions:
+
+    * `read_mf34`
+
+Function `read` reads a MF34/MT section from a string and produces a content
+object with a dictionary-like structure.
+The content object can be accessed using most of the keywords specified in
+the ENDF6 manual for this specific MF section.
+"""
+import sandy
+
+__author__ = "Aitor Bengoechea"
+
+
+def read_mf35(tape, mat, mt):
+    """
+    Parse MAT/MF=35/MT section from `sandy.Endf6` object and return
+    structured content in nested dcitionaries.
+
+    Parameters
+    ----------
+    tape : `sandy.Endf6`
+        endf6 object containing requested section
+    mat : `int`
+        MAT number
+    mt : `int`
+        MT number
+
+    Returns
+    -------
+    `dict`
+        Content of the ENDF-6 tape structured as nested `dict`.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> tape = sandy.get_endf6_file("jeff_33",'xs',922380)
+    >>> out = read_mf35(tape, mat=9237, mt=18)["SUB"][1]["FKK"][0:15]
```

Comment:
The actual test would consist of reading information from one text, writing it in another text and comparing the two. For this, I have already created the function, but it is necessary to see where it would be introduced.